Documentation/compatibility/ASPNET Accessibility Improvement.md

```diff
@@ -0,0 +1,47 @@
+## ASP.NET Accessibility Improvement in .NET 4.7.1
+
+### Scope
+Minor
+
+
+### Version Introduced
+.NET 4.7.1
+
+
+### Source Analyzer Status
+NotPlanned
+
+
+### Change Description
+ASP.NET is improving how ASP.NET Web Controls work with accessibility technology in Visual Studio to better support ASP.NET customers.  These includ the following changes:
+- Changes to implement missing UI accessbility patters in controls, like the Add Field dialog in Details View wizard.
+- Changes to improve display in High Contrast mode, like the Data Pager Fields Editor.
+- Improve the keyboard navigation experiences for controls, like Configure Object Context Window or Configure Data Source Window, etc.
+
```

Comment:
I think that this needs the following added:
- [X] Quirked
- [ ] Build-time break